.coveralls.yml

```diff
@@ -1 +0,0 @@
-repo_token: qpyDcfzamRoA7NkghlM1S8yLUk8H7Xd8U
```

Comment:
Did you generate another one, if it needs to be secret? :D

Reply:
@linuxbandit well I've switched from Coveralls to Codecov.io (that's why I don't need this one) and Codecov.io doesn't require repository token if it is running inside Travis